integrations/opensearch/src/haystack_integrations/document_stores/opensearch/document_store.py

```diff
@@ -44,6 +45,10 @@ def __init__(
         mappings: Optional[Dict[str, Any]] = None,
         settings: Optional[Dict[str, Any]] = DEFAULT_SETTINGS,
         create_index: bool = True,
+        http_auth: Any = None,
```

Comment:
I'm fine with that. Just want to mention technically it could be anything that whatever the connection class specified takes as `http_auth` param. That's why opensearch-py also typed it as `Any` I guess. E.g. https://github.com/opensearch-project/opensearch-py/blob/a68e8b7daff4312e8c220cd531ab9a2e5f2ccf9e/opensearchpy/connection/http_urllib3.py#L108
This connection class checks for Callable, tuple and list for example:
https://github.com/opensearch-project/opensearch-py/blob/a68e8b7daff4312e8c220cd531ab9a2e5f2ccf9e/opensearchpy/connection/http_urllib3.py#L141-L149

Reply:
Ok, cool. Let's leave it as `Any` then.